bool CloseGripper  
float64 PositionJoint1 
---
bool success